src/Microsoft.DocAsCode.Dfm/DfmCodeExtractor.cs

```diff
@@ -233,42 +233,81 @@ public DfmExtractCodeResult ExtractFencesCode(DfmFencesBlockToken token, string
 
                 return GetFencesCodeCore(fencesCode, resolveResult.StartLine, resolveResult.EndLine, resolveResult.IndentLength, resolveResult.ExcludesLines);
             }
-            else
+            else if (token.PathQueryOption?.Regions != null)
```

Comment:
abstract start-end, range to a method returns included lines with type IEnumerable<int>?